browser/resources/brave_new_tab_page_refresh/components/news/news_feed.tsx

```diff
@@ -22,6 +22,12 @@ export function NewsFeed() {
     setTimeout(() => setShouldRenderNews(true), 1000)
   }, [])
 
+  React.useEffect(() => {
+    const handleScroll = () => braveNews.setShouldRenderImages(true)
```

Comment:
no point attaching these handlers if `shouldRenderImages` is true
```suggestion
    if (braveNews.shouldRenderImages) {
        return
    }
    const handleScroll = () => braveNews.setShouldRenderImages(true)
```